### ISSUE TITLE
Document compatible apps

### ISSUE THREAD

**Issue by hobarrera** [pimutils/todoman#101]
Being standards-complaint, `todo` is compatible with lots of other apps (I hope!). I'd like to document and mention them, so users can know ahead of time if it's useful for them (mostly for nice PR really).

Currently, I know we're compatible with the iOS reminders app, and that's really the end of my list. Fastmail's calendar doesn't have reminders. :(

**Comment by savitha97** [comment]
I would like to work on this. Could you tell me where to start?

**Comment by untitaker** [comment]
@savitha97 For this task there's nothing to code. You'd have to find out which devices synchronize task lists using CalDAV (OpenTasks for Android and iOS reminders for example) and put them into a new [rst file here](https://github.com/pimutils/todoman/tree/master/docs/source)

**Comment by savitha97** [comment]
Am I actually required to run the apps and check or are online sources sufficient?

**Comment by untitaker** [comment]
@savitha97 Doesn't matter tbh, though for those cases we should mention that we didn't try it.

OpenTasks and iOS does work though, I tried both.

**Comment by untitaker** [comment]
I assume you're here because of rgsoc?

**Comment by savitha97** [comment]
Yes.

**Comment by hobarrera** [comment]
TBH, this isn't a very good RGSoC issue, and my hope is that it's gradually improved over time.

That said, you're still free to work on it if you like.

**Comment by savitha97** [comment]
@hobarrera  I would like to work on this at least to get hang of the workflow even if it may not be a very good RGSoc issue. Also, issues tagged with RGSoC seem a bit daunting as I have only begun understanding the repository.
Are there any set of additional requirements (apart from the tools and languages) to apply for this organisation in RGSoC'17?

**Comment by hobarrera** [comment]
Yes, sorry, that wasn't very clear (I think I never wrote this anywhere 😅 ): Issues tagged RGSoC are the ones you can mention in your application you'd be working on during the SoC (since you need to mention a specific issue, IIRC). I don't expect them to be picked up *first*, since they're not-that-simple.

* There's a [small checklist](http://todoman.readthedocs.io/en/stable/contributing.html#patch-review-checklist) to keep in mind when submitting patches.
* We recommend you create a virtualenv and run `pip install -e .` to develop on.
* I recommend configuring your IDE/editor to run flake8 to check for style compliance.
* **Some** understanding of OOP is required.

But the main thing, as you said, are the tools and languages.

Also, thanks for your interest in `todoman`.

**Comment by savitha97** [comment]
@hobarrera Thanks.
Regarding this issue, is it alright if I create a new rst file say, `compatible_apps.rst` with a table containing the following columns 
1. Application Name
2. Platform/OS
3. Tested (Y/N)
4. Additional Notes

**Comment by hobarrera** [comment]
Yup. Basically that would do. We should also mark those that are **not** compatible so other know not to bother testing. :)
